hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -202,6 +224,9 @@ public SingleTransactionRecordBuilderImpl(
      * @return the transaction record
      */
     public SingleTransactionRecord build() {
+        if (customizer != null) {
+            transaction = customizer.apply(transaction);
+        }
```

Comment:
I don't see a unit test for this?  (e.g., with a non-identify `apply`)

Reply:
Made `customizer` non-null.

Reply:
That isn't what I meant really.  I meant: Only things I see tested are where the customizer is the identity function or the suppressor.  There isn't a direct test with a customizer that does something non-trivial.  (There's a test where the cryptocreate gets changed to a contractcreate - I think I saw that yesterday though can't find it now - but that's rather indirect and I can't find a _direct_ test of this, which should probably be in `SingleTransactionRecordBuilderTest`.)